projects/kit/components/input-date/input-date.directive.ts

```diff
@@ -21,6 +22,10 @@ export class TuiInputDateDirective extends AbstractTuiTextfieldHost<TuiInputDate
         return this.host.computedMin;
     }
 
+    public get format(): TuiDateFormatSettings {
```

Comment:
With the decorator I spoke of it would be clean to reuse it inside `TuiNativeDateDirective` instead of introducing more coupling.

Reply:
agree